### PR TITLE
assignment-unit-3.1

### PR DESCRIPTION
**Pull request by Javierna182**

### CODE REVIEW

File: assignment/scripts/array-practice.js [hunk 1]
```diff
@@ -9,37 +9,40 @@ console.log('Animals are: ', animalArray);
...
+console.log('Second animal is', animalArray[1]);
 // 3.b. TODO: Log the last animal in the array using it's array index 
 
+console.log('Fourth animal is', animalArray[3]);
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
 
-
+console.log('last animal' ,[animalArray.length -1]);
```
Comment (delventhalz) [inline assignment/scripts/array-practice.js:45]: Not quite. I think you maybe meant to `animalArray` in front of the square brackets.

File: assignment/scripts/array-practice.js [hunk 2]
```diff
@@ -76,19 +88,39 @@ console.log('The animals are now', animalArray);
...
 //      with another one of your favorite foods.
 //      Then log the updated array.
 
+foodsArray [1] = 'pasta';
+console.log('My favorite foods are now:', foodsArray);
+
 // 4.f (STRETCH) TODO: Sort your favoriteFoods array
 //     in reverse alphabetical order. Log the array.
 
+const reversed = foodsArray.reverse();
```
Comment (delventhalz) [inline assignment/scripts/array-practice.js:105]: Take another look at the prompt. The assignment was to _sort_ the array.

File: assignment/scripts/array-practice.js [hunk 3]
```diff
@@ -76,19 +88,39 @@ console.log('The animals are now', animalArray);
...
+console.log('My favorite foods in reversed:', reversed);
+
 // 4.g (STRETCH) TODO: Convert your array to a string
 //     putting the word "and" between each item.
 //     eg "tacos and pizza and pasta". Log the string.
 
+const elements = foodsArray;
+
+console.log(elements.join('-'));
```
Comment (delventhalz) [inline assignment/scripts/array-practice.js:114]: You used the right array method, but the prompt was to put `" and "` between the foods, not `"-"`!

File: assignment/scripts/array-practice.js [hunk 4]
```diff
@@ -76,19 +88,39 @@ console.log('The animals are now', animalArray);
...
 // 4.h (STRETCH) TODO: Make a new array that combines 
 //     the favorite foods array with the animals array.
 //     Then log the new array.
 //     It should look something like:
-//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
\ No newline at end of file
+//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
+
+
+let bothArray = foodsArray.concat(animalArray);
```
Comment (delventhalz) [inline assignment/scripts/array-practice.js:125]: Nice. Could also have used the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) for this.

**Review by delventhalz — COMMENTED** [review]
Nice job, but make sure you carefully read those prompts and test your output in the console to make sure it is correct!

**Comment by Javierna182** [comment]
Zac for all the feedback!!

**Pull request closed without merging**